plugins/out_loki/loki.c

```diff
@@ -1200,9 +1200,11 @@ static void cb_loki_flush(const void *data, size_t bytes,
     /* User Agent */
     flb_http_add_header(c, "User-Agent", 10, "Fluent-Bit", 10);
 
-    /* Basic Auth headers */
-    if (ctx->http_user && ctx->http_passwd) {
+    /* Auth headers */
+    if (ctx->http_user && ctx->http_passwd) { /* Basic */
         flb_http_basic_auth(c, ctx->http_user, ctx->http_passwd);
+    } else if (ctx->bearer_token) { /* Bearer token */
```

Comment:
Could you please change this conditional to be `} else if (ctx->bearer_token != NULL) {`? If you did the same with the one above for consistency it would be really cool.